{{ cookiecutter.repo_name }}/.github/workflows/check_future_version.yml

```diff
@@ -0,0 +1,22 @@
+name: Check New Version
+
+on: workflow_dispatch
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    concurrency: release
+    permissions:
+      id-token: write
+      contents: write
+
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: Python Semantic Release
+        uses: python-semantic-release/python-semantic-release@master
+        with:
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+          root_options: "-v --noop"
```

Comment:
I would not mix many changes in a single PR - this one should focus on buliding options, not adding some Python Semantic-related changes

Reply:
The thing is this stuff is ready and also it seems normal that while making versioning optional we added another option for it.

Reply:
Done. All _Python Semantic Release_ stuff is not in #24